clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -4385,6 +4385,11 @@ public FenceProducersResult fenceProducers(Collection<String> transactionalIds,
         return new FenceProducersResult(future.all());
     }
 
+    @Override
+    public Uuid clientInstanceId(Duration timeout) {
+        throw new UnsupportedOperationException();
```

Comment:
nit: `UnsupportedOperationException("Not implemented yet.");`

Reply:
Other than test files I do not see other classes writing additional message hence omitted.